### ISSUE TITLE
'RTCPeerConnection': Malformed URL

### ISSUE THREAD

**Issue by brylie** [RocketChat/Rocket.Chat#2259]
When starting a video chat, the other participants are not visible. 

# Steps to reproduce
1. Sign in to Rocket Chat
2. Click on a participant name to open direct message
3. Click "User Info" tab in right sidebar
4. Click "Video" button
5. Grant audio/video permissions in browser
6. Accept video chat invitation in second account

# Observed behavior
Everything works locally:
* invitation is sent
* invitation is received
* video and audio permissions are granted on both clients
* local video and name displays in chat area for each client

However, the second participant video and name does not appear.

## Console errors
As soon as the invited participant clicks "Accept", the following errors appear in the console of the originating user:

```
Exception in queued task: TypeError: Failed to construct 'RTCPeerConnection': Malformed URL
    at TypeError (native)
    at n.getPeerConnection (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:317:8186)
    at n.onRemoteJoin (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:317:15635)
    at e.onUserStream (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:317:1982)
    at Object.<anonymous> (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:317:18308)
    at Object.n (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:160:297)
    at Object.r.find.observe.added (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:160:935)
    at a.added (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:62:4063)
    at Object.n.ordered.n.applyChange.added (https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:62:2711)
    at https://chat.samposoftware.com/90eb095829ef4ecc01915a8bbdcb1ae51e8ae8de.js?meteor_js_resource=true:60:4003
``` 

# Expected behavior
The second participant should show up in the video list. Second participant video and name should be prominently displayed.

# Environment
* Rocket Chat 0.18.1
* Browsers: Firefox and Chromium
* Server: Ubuntu 14.04
* Installation method: [Docker installation](https://github.com/RocketChat/Rocket.Chat/wiki/Docker---Ubuntu-with-Nginx-SSL-and-Hubot)
* SSL: we enabled ssl and try to ensure no mixed content (such as favicons) are served over HTTP
* Network: two laptops tested on both separate and the same wi-fi network(s)
* WebRTC settings:
![screenshot-1](https://cloud.githubusercontent.com/assets/17307/13107003/059447d4-d574-11e5-936d-d27f75cd256b.png)

**Comment by rodrigok** [comment]
Fixed via #2267

**Comment by brylie** [comment]
OK, thanks :-) We will await the 0.19 release.
